lib/sensu-plugins-mongodb/metrics.rb

```diff
@@ -102,6 +102,26 @@ def replicaset_status
       end
     end
 
+    def wiretiger_metrics(wiredtiger, server_metrics)
+      # wiredtiger = server_status['wiredTiger']
+      # data_handle Metrics
+      wiredtiger.each do |key1, value1|
+        next unless value1.is_a? Hash
+        wiredtiger[key1].each do |key2, value2|
+          if value2.is_a? Hash
+            wiredtiger[key1][key2].each do |key3, value3|
+              server_metrics['wiredTiger.' + key1.gsub(/(,|\s|-|\()+/, '_').tr(')', '') + '.' +
```

Comment:
it looks like we are repeating the same logic/regex here can we make a function and call that instead I think it would make the code a lot more readable. Maybe something along the lines of `wiretiger_metric_(converter|builder|formatter)`

Reply:
I was thinking to write json flattener but current use case was quite specific so I quickly submit this PR. But I kinda liked the idea.